sdcm/nemesis.py

```diff
@@ -527,6 +527,23 @@ def disrupt_nodetool_enospc(self, sleep_time=30, all_nodes=False):
             nodes = [self.target_node]
         self._set_current_disruption('Enospc test on {}'.format([n.name for n in nodes]))
 
+        for node in nodes:
+            with DbEventsFilter(type='NO_SPACE_ERROR', node=node), \
+                    DbEventsFilter(type='BACKTRACE', line='No space left on device', node=node), \
+                    DbEventsFilter(type='DATABASE_ERROR', line='No space left on device', node=node), \
+                    DbEventsFilter(type='FILESYSTEM_ERROR', line='No space left on device', node=node):
+
+                result = node.remoter.run('cat /proc/mounts')
+                if '/var/lib/scylla' not in result.stdout:
+                    self.log.error("Scylla doesn't use an individual storage, skip enospc test")
+                    continue
+
+                self.reach_enospc_on_node(target_node=node, log_object=self.log)
+
+                self.clean_enospc_on_node(target_node=node, log_object=self.log, sleep_time=sleep_time)
+
+    @staticmethod
+    def reach_enospc_on_node(target_node, log_object):
```

Comment:
from the architecture point of view passing, `log_object` is not a good idea!
better to get his methods out to utils.py and use regular LOGGER